app/server/src/main/scala/org/bitcoins/server/BitcoindRpcBackendUtil.scala

```diff
@@ -217,32 +206,27 @@ object BitcoindRpcBackendUtil extends Logging {
     pairedWallet
   }
 
-  private def filterSync(
-      blockHashes: Vector[DoubleSha256Digest],
+  private def filterSyncSink(
       bitcoindRpcClient: V19BlockFilterRpc,
-      wallet: Wallet)(implicit system: ActorSystem): Future[Unit] = {
+      wallet: Wallet)(implicit
+      system: ActorSystem): Sink[DoubleSha256Digest, Future[Wallet]] = {
```

Comment:
note: Returning `Sink` here now rather than the materialized stream represented by `Future[Unit]`. This allows us to connect this `Sink` to other stream pieces.